### ISSUE TITLE
sleep mode

### ISSUE THREAD

**Issue by zeud** [hasankhan/Squiggle#4]
How to fix crash after sleep mode? please

**Comment by cocoon** [comment]
Yeah I had same problems, I did a sort of dirty workaround but it is working at least for me.
No crashes since more than one week and no strange behavior, memory usage stays low between 22MB and 35.

But I did some more changes like:
Replaced NetMQ with NetMQ v4 source
modified popups to be shown always if the chatbox is not focused
added unfinished multimonitor popups, working for me, but needs fixes if main screen is not the first left one, it wil use negative values for width

**Comment by zeud** [comment]
when i update NetMQ  to NetMQ  4, i have many errors
please can you send me source code with these changes?
zeud3891@hotmail.com
